server/views/applications/submissionConfirmation.njk

```diff
@@ -0,0 +1,43 @@
+{% from "govuk/components/summary-list/macro.njk" import govukSummaryList %}
+{% from "govuk/components/panel/macro.njk" import govukPanel %}
+{% from "govuk/components/button/macro.njk" import govukButton %}
+
+{% extends "../partials/layout.njk" %}
+
+{% set pageTitle = applicationName + " - " + pageHeading %}
+{% set mainClasses = "app-container govuk-body" %}
+
+{% block content %}
+
+    <div class="govuk-grid-row">
+        <div class="govuk-grid-column-two-thirds">
+            {{ govukPanel({
+                titleText: "Application confirmation"
+            }) }}
+            <p>
+          We’ve sent you a confirmation email.
+        </p>
+            <p>
+                <a href="">Download a copy of this application (PDF, 708 KB) </a>
```

Comment:
I missed this from the prototypes. Will be very cool to be able to do this. Should be fairly easy too!